backend/app/models/chat_room_user.rb

```diff
@@ -1,4 +1,11 @@
 class ChatRoomUser < ApplicationRecord
   belongs_to :user
   belongs_to :chat_room
+
+  def increment_chat_count_and_check_payment
```

Comment:
例外が発生する可能性があるメソッドは!をつけましょう

```suggestion
  def increment_chat_count_and_check_payment!
```